packages/cli/src/lib/mini-oxygen/workerd.ts

```diff
@@ -262,10 +262,21 @@ async function miniOxygenHandler(
   return response;
 }
 
-function createAssetHandler(assetsPort: number) {
+function createAssetHandler(assetsPort: number, publicStoreDomain?: string) {
   const assetsServerOrigin = buildAssetsUrl(assetsPort);
 
   return async (request: Request): Promise<Response> => {
+    const {pathname} = new URL(request.url);
+
+    const wpmAssets = pathname.startsWith('/wpm@') || pathname.startsWith('/cdn/wpm/');
+    if (publicStoreDomain && wpmAssets) {
+      return fetch(
+        `https://${publicStoreDomain}/${pathname}`, {
+          signal: request.signal,
+          headers: request.headers,
+      });
+    }
```

Comment:
Interesting, I assume this is the proxy to all the web pixel assets stored on the online store domain?